### PR TITLE
Add intro and installation topics

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds the intro and installation topics to the Admin UI SDK guide.

## Affected pages

- ...

### CODE REVIEW

File: src/pages/admin-ui-sdk/installation.md [hunk 1]
```diff
@@ -1,6 +1,92 @@
...
-# Installation
+# Install Adobe Commerce Admin UI SDK
+
+This topic describes the basic steps you need to install the Admin UI SDK.
+
+## Prerequisites
+
+### Adobe Commerce
+
+* Adobe Commerce on cloud infrastructure or on premises: 2.4.5+
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/installation.md:14]: Did we double check that the AdminIMS module is available for 2.4.5?

Comment (keharper) [reply src/pages/admin-ui-sdk/installation.md:14]: IMS support is mentioned in the 2.4.5 release notes

https://experienceleague.adobe.com/docs/commerce-operations/release/notes/adobe-commerce/2-4-5.html?lang=en#adobe-commerce-integration-with-adobe-ims

File: src/pages/admin-ui-sdk/index.md [hunk 2]
```diff
@@ -4,3 +4,25 @@ description: An architectural overview of the Adobe Commerce Admin UI SDK
 ---
 
 # Adobe Commerce Admin UI SDK overview
+
+The Adobe Commerce Admin UI SDK enables an Adobe App Builder developer to extend the Commerce Admin to include custom menus and pages. These Admin customizations were traditionally defined in a PHP module. The principles of Out of Process development require a more modern approach. The Admin UI SDK provides the merchant with the means to use the Admin to configure and interact with an app developed with App Builder.
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/index.md:8]:
```suggestion
The Adobe Commerce Admin UI SDK enables an Adobe App Builder developer to extend the Commerce Admin to include custom menus and pages. These Admin customizations were traditionally defined in a PHP module. The principles of Out-of-Process development require a more modern approach. The Admin UI SDK provides the merchant with the means to use the Admin to configure and interact with an app developed in App Builder.
```

Comment (jhadobe) [reply src/pages/admin-ui-sdk/index.md:8]: are we not hyphenating phrases like "Out of Process UI Extensibility"? etc

File: src/pages/admin-ui-sdk/installation.md [hunk 3]
```diff
@@ -1,6 +1,92 @@
...
-title: Installation
-description: 
+title: Install Adobe Commerce Admin UI SDK
+description: Learn how to install the Adobe Commerce Admin UI SDK.
 ---
 
-# Installation
+# Install Adobe Commerce Admin UI SDK
+
+This topic describes the basic steps you need to install the Admin UI SDK.
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/installation.md:8]:
```suggestion
This topic describes the basic steps to install the Admin UI SDK.
```

File: src/pages/admin-ui-sdk/installation.md [hunk 4]
```diff
@@ -1,6 +1,92 @@
...
+
+The [Adobe App Builder _Getting Started_ guide](https://developer.adobe.com/app-builder/docs/getting_started/) lists the latest software requirements for creating App Builder apps.
+
+## Installation steps
+
+You can install the Admin UI SDK on Adobe Commerce on cloud infrastructure and on-premises instances.
+
+### Adobe Commerce on cloud infrastructure
+
+This method is used for installing the SDK for a cloud instance.
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/installation.md:32]:
```suggestion
This method installs the SDK on a cloud instance.
```
Not sure how the SDK works, so not sure if `on` is appropriate here.

File: src/pages/admin-ui-sdk/installation.md [hunk 5]
```diff
@@ -1,6 +1,92 @@
...
+
+   ```bash
+   composer update magento/module-commerce-backend-uix
+   ```
+
+1. Commit and push your changes.
+
+### On-premises
+
+This method is used for installing the SDK for an On-premises instance.
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/installation.md:58]:
```suggestion
This method installs the SDK on an On-premises instance.
```

File: src/pages/admin-ui-sdk/installation.md [hunk 6]
```diff
@@ -1,6 +1,92 @@
...
+   bin/magento setup:upgrade
+   ```
+
+1. Clear the cache:
+
+   ```bash
+   bin/magento cache:clean
+   ```
+
+1. Commit changes.
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/installation.md:90]:
```suggestion
1. Commit your changes.
```

**Review by jhadobe — APPROVED** [review]
Approved with a few small suggestions.

File: src/pages/admin-ui-sdk/index.md [hunk 7]
```diff
@@ -4,3 +4,25 @@ description: An architectural overview of the Adobe Commerce Admin UI SDK
...
+
+![Architecture](../_images/admin-ui-sdk-architecture.svg)
+
+The SDK contains two main components that are necessary for development:
+
+* The `CommerceBackendUix` module is a traditional in-process Commerce PHP module that allows Out of Process extensions to inject menus and pages into the Admin.
+
+* `uix` defines a React app that renders the custom Admin content.
+
+[Adobe Identity Management Service (IMS)](https://experienceleague.adobe.com/docs/commerce-admin/start/admin/ims/adobe-ims-integration-overview.html) manages authentication. The merchant adds their IMS org information into a configuration panel in Commerce. Then an app, which is installed in merchant's App Builder can securely communicate with the merchant's instance and render the app UI.
```
Comment (naomirobichaud) [inline src/pages/admin-ui-sdk/index.md:24]: Nitpicky comment but on line 24, suggest either removing first comma or an additional comma after App Builder.

**Review by asalloum5 — APPROVED** [review]

**Pull request merged**